pvt_cleaning.ipynb

```diff
@@ -0,0 +1,1582 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #71.                if not isinstance(cat_vector_for_week, str):
</pre><p>Is this needed to fill in data for the years before a category started being records?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>